### PR TITLE
List uPlexa (UPX)

### PR DESCRIPTION
**Pull request by QuantumLeaper**
- Official project URL: https://uplexa.com
- Official block explorer URL: https://explorer.uplexa.com

Additional: UPX wallets support transaction keys that enable 3rd party audits in case of a dispute similar to Monero

Included: Support for standard addresses (UPX), sub-addresses (UmV & UmW), and integrated address (UPi)

### CODE REVIEW

**Comment by blabno** [comment]
If this is privacy coin you have to warn users (when they create payment account) that only transparent addresses are allowed. Have a look how it's done here: https://github.com/bisq-network/bisq/pull/2433/files

**Comment by QuantumLeaper** [comment]
Thanks @blabno, I have gone a head and added the warning regarding TX verification/proof and squashed to one commit.

Cheers

File: assets/src/main/java/bisq/asset/coins/uPlexa.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.asset.coins;
+
+import bisq.asset.Coin;
+import bisq.asset.RegexAddressValidator;
+
+public class uPlexa extends Coin {
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/uPlexa.java:25]: Add `AltCoinAccountDisclaimer` annotation with proper key.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

File: assets/src/main/java/bisq/asset/coins/uPlexa.java (hunk 1 above)
Comment (QuantumLeaper) [reply assets/src/main/java/bisq/asset/coins/uPlexa.java:25]: My apologies for missing that. Updated & tested.

**Review by blabno — APPROVED** [review]
ACK

**Pull request merged**